src/applications/claims-status/tests/components/ClaimComplete.unit.spec.jsx

```diff
@@ -1,17 +1,22 @@
 import React from 'react';
-import SkinDeep from 'skin-deep';
 import { expect } from 'chai';
+import { render } from '@testing-library/react';
+import { $ } from '@department-of-veterans-affairs/platform-forms-system/ui';
 
 import ClaimComplete from '../../components/ClaimComplete';
 
 describe('<ClaimComplete>', () => {
   it('should render message', () => {
     const date = '2010-09-01';
-    const tree = SkinDeep.shallowRender(<ClaimComplete completedDate={date} />);
-
-    expect(tree.text()).to.contain(
-      'We decided your claim on September 1, 2010',
+    const { container, getByText } = render(
+      <ClaimComplete completedDate={date} />,
     );
-    expect(tree.everySubTree('CompleteDetails')).to.not.be.false;
+    const paymentsSection = $('.payments-container', container);
+    expect(paymentsSection).to.exist;
+
+    const text1 =
```

Comment:
Is there a text2?

Reply:
no there is not! Updated to just be text @samcoforma